src/main/java/org/alfresco/mockeventgenerator/EventSender.java

```diff
@@ -57,7 +57,7 @@ public void sendRandomEvent(int numOfEvents, long pauseTimeMillis)
     {
         for (int i = 0; i < numOfEvents; i++)
         {
-            sendEvent(eventTypeCategory.getRandomEvent());
+            sendEvent(eventTypeCategory.getRandomEvent(), "");
```

Comment:
There should be an overloaded version of this method that doesn't need the endpoint parameter.